runtime/autoload/health/nvim.vim

```diff
@@ -24,6 +24,19 @@ function! s:check_config() abort
           \   'Check `:verbose set paste?` to see if a plugin or script set the option.', ])
   endif
 
+  if !filereadable($HOME.'/.config/nvim/init.vim') && filereadable($HOME.'/.vim/vimrc') && filereadable($HOME.'/.vimrc')
+    let ok = v:false
+    call health#report_warn("It seems you are new to neovim but used vim.",
```

Comment:
"Vim" is usually capitalized. So is "Neovim", but you should use "Nvim" instead, like in the other health checks. Here and below.